docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/psa-config-templates.md

```diff
@@ -106,6 +107,8 @@ When you run Rancher on a Kubernetes cluster that enforces a restrictive securit
 - `cattle-monitoring-system`
 - `cattle-neuvector-system`
 - `cattle-prometheus`
+- `cattle-provisioning-capi-system`
+- `cattle-resources-system`
```

Comment:
Why is `cattle-resources-system` also being added to exemptions ns for PSA? Applies to all references in the PR.

Reply:
`cattle-resources-system` is used by the app rancher-backup-restore-operator which is used in restoring or migrating Rnacher on a hardened cluster, so we need to include it in the exception list.